src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/IntegrationTestBase.java

```diff
@@ -45,4 +70,50 @@ public void setup () {
         entityGenerator = new EntityGenerator(injector.getInstance(OctaneProvider.class));
     }
 
+    @Test
+    public void createWorkSpace() {
```

Comment:
I see you've made the createWorkSpace method, that's cool, but it shouldn't be a test. it's a private utility method that may or may not be used.